datasets/ogbg-molbace/ogbg-molbace_task.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Name this file as task_graph_classification.json. Same for all other datasets.

Reply:
done.